docs/_docs/usage/coverage.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: doc-page
+title: "Code Coverage for Scala 3"
+---
+
+## Instrument code for coverage analysis
+
+[PR#13880](https://github.com/lampepfl/dotty/pull/13880) has implemented code coverage support for Dotty.
+In general, code coverage works in three steps:
+1. The program is "instrumented" at compilation time: code is inserted to record which statement are called. This does not change the behavior of the program. Also, a list of all the coverable statements is produced.
+2. The program is run, usually by unit tests, and the instrumentation code saves its measurements.
+3. Some tool processes the data to generate a fancy coverage report, for instance a web page.
+
+In Scala 2, all these steps were performed by external tools. In particular, step 1 was implemented by a compiler plugin.
+
+In Scala 3, the compiler itself takes care of step 1. To use this feature, add the compile option `-coverage-out:DIR`, where `DIR` is the destination of the measurement files.
+
+You can also set `-coverage-sourceroot:PATHS_ROOT` to customize how the path of your source files are resolved.
```

Comment:
Can the existing -sourceroot be used for this purpose instead or is there a difference between the two?

Reply:
Hmmm
Maybe `sourcepath`, but I would rather keep a separate setting for the coverage. It's not meant to indicate to the compiler where the files are, but to generate relative paths in the scoverage reports. This is good to have when external tools that consume the reports (Sonarqube and co.) don't always understand the project layout or are started (by a CI) in an unexpected directory.

Reply:
sourceroot is similar but used to relativize paths stored in tasty files

Reply:
oh! in that case yes, why not